### ISSUE TITLE
Keyboard navigation doesn't work as expected on TextInput

### ISSUE THREAD

**Issue by gen100mek** [VirtusLab/tetrisly-react#119]
Version: 1.5.0

When user has a keyboard focus on a `TextInput`, there's no way to focus a previous item in focus order by pressing `Shift+Tab`. It also works the same on Storybook: https://storybook.tetrisly.com/?path=/docs/textinput--docs
